forge-gui/res/cardsfolder/upcoming/flames_of_moradin.txt

```diff
@@ -0,0 +1,11 @@
+Name:Flames of Moradin
+ManaCost:2 R R
+Types:Sorcery
+A:SP$ Destroy | TargetMin$ 0 | TargetMax$ 3  | Cost$ 3 R | ValidTgts$ Artifact | SubAbility$ DBRepeatEach | SpellDescription$ Destroy up to three target artifacts. Conjure a duplicate of each nontoken artifact destroyed this way into your hand. The duplicates perpetually gain “You may pay {R} rather than pay this spell's mana cost" and "at the beginning of your end step, sacrifice this artifact."
+SVar:DBRepeatEach:DB$ RepeatEach | RepeatSubAbility$ DBConjure | RepeatCards$ Card.Artifact+IsRemembered
+SVar:DBConjure:DB$ MakeCard | ConditionDefined$ Remembered | ConditionPresent$ Card | DefinedName$ Remembered.nonToken | Zone$ Hand | SubAbility$ DBEffect

```

Comment:
I think `ConditionPresent$ Card.nonToken | DefinedName$ Remembered` would be better here